symla/kronecker.py

```diff
@@ -33,11 +29,12 @@ def is_zero(x):
 class FiniteVectorSpace(Basic):
     """
     """
-    def __new__(cls, name, shape=None):
+    def __new__(cls, name, dimension=None, field=None):
 
```

Comment:
Should we inform the dimension and field of a vector space? or should we inject that information later?
I think dimension should be used here instead of shape as it is the case in Psydac.